### DISCUSSION TITLE
PR from fork does not show actions even though actions start on the fork

### DISCUSSION THREAD

**Discussion by phw** [community/community#25099] [Actions]
<p>I have opened a PR from my fork of a repository to the upstream repository. Both repositories can run Github Actions. But the PR itself does not show any information on the checks.</p>
<p>This is the PR:</p>
<p><a href="https://github.com/metabrainz/picard/pull/1376" rel="nofollow noopener">https://github.com/metabrainz/picard/pull/1376</a></p>
<p>As you see the checks are empty. But on the forked repository I see the actions being executed: <a href="https://github.com/phw/picard/commit/59b88f41e6e9502c08d033c1a9b4e82de6ec36be/checks?check_suite_id=333414038" rel="nofollow noopener">https://github.com/phw/picard/commit/59b88f41e6e9502c08d033c1a9b4e82de6ec36be/checks?check_suite_id=333414038</a></p>
<p>As I understood Actions so far, the actions should be executed on the fork for security reason, but they should still show up in the PR. This is how other CI systems, e.g. Travis or Appveyor, also behave.</p>

**Comment by lannonbr** [comment]
<p>I’ve been running into this as well. Curious if there is any plans to make them appear in a checksuite in the future</p>

**Comment by phw** [comment, marked as answer]
<p>I forgot to answer here, but I actually figured it out. You need to run the workflow on pull_request, e.g.</p>
<p>on: [push, pull_request]</p>
